### PR TITLE
Lint message softening.

### PR DESCRIPTION
**Pull request by pq**
@lukechurch : for your enjoyment.

### CODE REVIEW

File: lib/src/rules/empty_constructor_bodies.dart [hunk 1]
```diff
@@ -7,7 +7,7 @@ library linter.src.rules.empty_constructor_bodies;
 import 'package:analyzer/src/generated/ast.dart';
 import 'package:linter/src/linter.dart';
 
-const desc = 'DO use ; instead of {} for empty constructor bodies.';
+const desc = 'Use ; instead of {} for empty constructor bodies.';
```
Comment (lukechurch) [inline lib/src/rules/empty_constructor_bodies.dart]: ';'

File: lib/src/rules/non_constant_identifier_names.dart [hunk 2]
```diff
@@ -8,7 +8,7 @@ import 'package:analyzer/src/generated/ast.dart';
 import 'package:linter/src/linter.dart';
 import 'package:linter/src/util.dart';
 
-const desc = r'DO name non-constant identifiers using lowerCamelCase.';
+const desc = r'Name non-constant identifiers using lowerCamelCase.';
```
Comment (lukechurch) [inline lib/src/rules/non_constant_identifier_names.dart]: Do we need `lowerCamelcase` here?

File: lib/src/rules/type_init_formals.dart [hunk 3]
```diff
@@ -7,7 +7,7 @@ library linter.src.rules.type_init_formals;
 import 'package:analyzer/src/generated/ast.dart';
 import 'package:linter/src/linter.dart';
 
-const desc = "DON'T type annotate initializing formals";
+const desc = "Don't type annotate initializing formals";
```
Comment (lukechurch) [inline lib/src/rules/type_init_formals.dart]: Consistency with trailing '.' with the above. These should either all be using closing punctuation or none of them.

**Comment by lukechurch** [comment]
lgtm with nits.

File: lib/src/rules/type_init_formals.dart (hunk 3 above)
Comment (pq) [inline lib/src/rules/type_init_formals.dart]: Well, you know what they say about consistency...  But Ok!

**Comment by pq** [comment]
Thanks!

File: lib/src/rules/empty_constructor_bodies.dart (hunk 1 above)
Comment (pq) [inline lib/src/rules/empty_constructor_bodies.dart]: :+1:

**Pull request merged**